services/blockchain-connector/shared/sdk/dynamicReward.js

```diff
@@ -36,6 +40,22 @@ const getRewardTokenID = async () => {
 	return rewardTokenID;
 };
 
+const getDefaultRewardAtHeight = async height => {
+	try {
+		// TODO: Update endpoint once exposed by sdk
```

Comment:
Do we still need the TODO? 🤔 
```suggestion
```

Reply:
Better to keep it as the endpoint is still not running.